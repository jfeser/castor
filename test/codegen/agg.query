select([(1.0 + 2.0) as x, avg(a) as y, count() as z, sum(a) as r, min(a) as s, max(a) as t],
  alist(r2, ascalar(0.a)))