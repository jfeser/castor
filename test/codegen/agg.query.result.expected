3.000000|7.978000|5|39.890000|-0.420000|34.420000
